Assets/Scripts/Tools/FlyTool.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System.Collections;
```

Comment:
Can you please add a header, similar to that of the other files, with `Copyright (c) 2021 The Open Brush Authors` in place of the Google/TiltBrush copyright? 

After that, we're ready to merge this

Reply:
Done - I literally just copied the existing license comment, which I'm assuming is right?